collaboration/01-sig-and-wg.md

```diff
@@ -16,40 +16,41 @@ WGs facilitate discussions and work on short-lived, concrete topics that either
 
 While most of the work within one SIG or WG group does not require cooperation with other SIGs, there are efforts, such as larger feature development and refactoring, that cross a single SIG's boundaries. In this case, the members of involved SIGs collaborate and agree on solutions.
 
-The cross-project activities can require a creation of a new WG to address them more specifically. Those overarching activities produce coordination overhead with regards to time and efforts.
+The cross-project activities can require the creation of a new WG to address them more specifically. Those overarching activities produce coordination overhead with regards to time and efforts.
 
 ## Kyma project organization
 
 The Kyma project is open-source and is available on [GitHub](https://github.com/kyma-project). Although external contributions to the Kyma open-source project are welcome through the public GitHub project, the Kyma teams within the organization largely drive the Kyma development.
 
-To ensure the continuous growth and the "scale-as-you-go-and-need" approach when extending the community, the current structure and governance bodies of Kyma SIGs and WGs are very simple. However, the structure of the community can change depending on the community needs. The SIG Community and the [Kyma Steering Committee](/governance/#kyma-steering-committee-kyma-steering-committee) support the governance of Kyma and a single SIG Core represents it publicly. The aim of the SIG Core is to facilitate the collaboration with external contributors.
+To ensure the continuous growth and the "scale-as-you-go-and-need" approach when extending the community, the current structure and governance bodies of Kyma SIGs and WGs are very simple. However, the structure of the community can change depending on the community needs. The [Kyma Steering Committee](/governance/#kyma-steering-committee-kyma-steering-committee) supports the governance of Kyma and a single SIG Core represents it publicly. The aim of the SIG Core is to facilitate the collaboration with external contributors.
```

Comment:
Actually, the [SIG&WG process](https://kyma-project.io/community/collaboration/#special-interest-group-working-group-special-interest-group-working-group) is rendered on our website, similarly to the doc on the [Steering Committee](https://kyma-project.io/community/governance/#kyma-steering-committee-kyma-steering-committee) to which it points. Our general approach is to make documentation readable from the website, (instead of GitHub sources) whenever possible, that's why this link needs to stay this way.